airflow/providers/salesforce/operators/salesforce_bulk.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we should just call it `bulk.py` ?
@josh-fell WDYT?

Reply:
I agree. Slightly unfortunate we already have `salesforce_apex_rest.py`. The naming is a little redundant and I don't think we need to continue to pattern.

Reply:
Renamed `salesforce_apex_rest.py` to `bulk.py`.